tests/wrapper.py

```diff
@@ -0,0 +1,42 @@
+import io
+import sys
+
+from decorator import decorator
+
+
+def wrap_cli_test(output, save_output=False):
+    """
+    This decorator captures the stdout and stder and compare it
+    with the contects of the specified files.
```

Comment:
Small typo "contects".

I've been thinking that we should perhaps have something like this in Brigade to test https://github.com/brigade-automation/brigade/blob/develop/brigade/plugins/functions/text/__init__.py, might also move the test coverage over 90% so the badge looks nicer :)

Reply:
Yeah, that's a good idea. Would you mind sending a PR?

Will fix the typo.